src/createMatcher.js

```diff
@@ -0,0 +1,9 @@
+import minimatch from 'minimatch';
+const path = require('path');
+
+export default pattern => {
+  if (typeof pattern !== 'string') {
```

Comment:
I'd have to double check, but if you do `pretty-quick --pattern a --pattern b`, does `mri` make `pattern` equal `['a', 'b']`?

Reply:
(and if so, should we handle it?)

Reply:
I think we should either (a) handle multiple patterns or (b) print a warning rather than ignoring multiple patterns. I have no strong feelings one way or the other :P

Reply:
I also don't mind keeping the `multimatch` dependency.

Reply:
I’ll take a look and let you know :)

Reply:
It supports now three cases (using multimatch)

  - no patterns
  - 1 pattern (string)
  - n pattern (array of strings)